hoomd/md/IntegratorTwoStep.cc

```diff
@@ -104,9 +104,13 @@ void IntegratorTwoStep::update(unsigned int timestep)
         m_prof->push("Integrate");
 
     // perform the first step of the integration on all groups
-    std::vector< std::shared_ptr<IntegrationMethodTwoStep> >::iterator method;
-    for (method = m_methods.begin(); method != m_methods.end(); ++method)
+    for (auto method = m_methods.begin(); method != m_methods.end(); ++method)
```

Comment:
We could use a range-based for loop here.

Reply:
Done.